Tools/autotest/common.py

```diff
@@ -829,14 +826,14 @@ def mavproxy_do_set_mode_via_command_long(self, mode, timeout=30):
         while True:
             remaining = timeout - (self.get_sim_time_cached() - tstart)
             if remaining <= 0:
-                raise AutoTestTimeoutException()
+                raise AutoTestTimeoutException("Failed to change mode")
             self.mavproxy.send("long DO_SET_MODE %u %u\n" %
                                (base_mode, custom_mode))
             m = self.mav.recv_match(type='HEARTBEAT',
                                     blocking=True,
-                                    timeout=remaining)
```

Comment:
Was this intentional? shouln't it be done in a separated comit?

Reply:
Probably should be a separate commit.  *shrugworthy* :-)